polyfills/WebAnimations/config.json

```diff
@@ -0,0 +1,22 @@
+{
+	"browsers": {
+		"ie": "10 - *",
+		"ie_mob": "10 - *",
+		"firefox": "27 - 50",
+		"opera": "11.6 - 42",
+		"safari": "9 - *",
+		"chrome": "* - 38",
+		"ios_saf": "7.1 - *",
+		"firefox_mob": "27 - 50",
+		"android": "<53"
+	},
+	"dependencies": [],
+	"license": "Apache-2.0",
+	"spec": "https://html.spec.whatwg.org/multipage/embedded-content.html#embedded-content",
+	"repo": "https://www.w3.org/TR/web-animations-1/",
```

Comment:
The `repo` field is used to specify the repository where the polyfill code can be found, it should be `https://github.com/web-animations/web-animations-js`